### PR TITLE
incose tutorial

### PR DESCRIPTION
**Pull request by jadelkhoury**

### CODE REVIEW

**Comment by jadelkhoury** [comment]
@berezovskyi @axelreichwein ! I would need your help to work out how I can make the pdf-download works.
this article has a Download link (https://open-services.net/resources/presentation-20171117/) but I don't seem to find the actual pdf in the repo. How can I upload the pdf to make it downloadable?
I have added the pdf in this commit.

**Comment by berezovskyi** [comment]
@jadelkhoury just fixed the link. You can test locally with `hugo server -w`.

I also fixed the OpenPDM link because it should not have been stored under the themes/ dir. Ideally, we'd move it under the /static/files/2015/ directory from /static/resources/ but that may break some links in Google.

@jadelkhoury @jamsden I would suggest you consider a https://open-services.net/news/ post instead or in addition to this resource entry because the news section has been a ghost town for 2 years.

File: content/resources/tutorial_-_Digital_Threads_with_OSLC.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+tags = ["tutorial"]
+title = "Digital threads with OSLC"
+summary = "An OSLC tutorial at the the 33rd Annual INCOSE International Symposium"
+
++++
+
+
+This tutorial was given at the 33rd Annual INCOSE International Symposium on July 16th.
+
+It was delivered by Erik Herzog (Technical Fellow, SAAB Aeronautics) and Eran Gery (Global Systems Engineering Solutions Lead, IBM). Contributions were also made by Jad El-khoury (LynxWorks), Ian Green (IBM) and David Honey (IBM).
```
Comment (berezovskyi) [inline content/resources/tutorial_-_Digital_Threads_with_OSLC.md:14]: Is it LynxWorks or Lynxwork?

**Comment by berezovskyi** [comment]
And @jadelkhoury, the CLA check fails because your first commit was made with another account. Here is https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-a-single-commit for you because I think this is not the last time you will make this mistake 🤣

**Review by berezovskyi — DISMISSED** [review]

**Review by berezovskyi — APPROVED** [review]

**Comment by jadelkhoury** [comment]
thanks @berezovskyi Still a mystery to me were the OpenPDM-pdf file resides, but happy you could help.

Regarding the News, I have otherwise opted to put something on the forum. Will consider.
and thanks for the tips about the commit author.

**Comment by berezovskyi** [comment]
@jadelkhoury, the OpenPDM PDF was under the static folder, in the theme resources. I moved it under general resources folder under static. Those two folders are merged by Hugo generator without further processing.

image

**Comment by jadelkhoury** [comment]
ok thanks andrii.

**Pull request merged**